backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/commons/tickets.js

```diff
@@ -54,7 +53,20 @@ const templateIDToParams = async (req, res, next) => {
 	}
 };
 
-TicketsMiddleware.validateNewTicket = validateMany([templateIDToParams, checkTicketTemplateExists, validateNewTicket]);
+const setSafetibaseModValues = (req, res, next) => {
+	const oldTicketSafetibaseMod = req.ticketData.modules?.safetibase;
+	const updatedTicketSafetibaseMod = req.body.modules?.safetibase;
+
+	if (oldTicketSafetibaseMod && updatedTicketSafetibaseMod) {
```

Comment:
because if someone want to update just one safetibase prop I want to retrieve all the rest of existing values to calculate levelOfRisk and treatedLevel... otherwise the param received in processReadOnlyValues will have only the prop to be updated